js-pkg/apps/demos/src/components/webrtc.js

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react'
+import { useDriftDBSignalingChannel } from './driftdbutils'
+
+export const useWebRTCMessagingChannel = (p1, p2) => {
+  let [setConnSetupArray] = useWebRTCConnection(p1, p2)
+  let [theyAreReady, setTheyAreReady] = React.useState(false)
+  let [messages, addMessage] = React.useReducer((state, msg) => [...state, msg], [])
+  let theirdataChannelRef = React.useRef(null)
+
+  React.useEffect(() => {
+    setConnSetupArray([
```

Comment:
From offline convo, maybe you could pass this list of functions to `useWebRTCConnection()`?